<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectDock</class>
 <widget class="QDockWidget" name="EffectDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>433</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Effect</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="EffectListView" name="listView"/>
    </item>
    <item>
     <widget class="EffectListView" name="listView_2"/>
    </item>
   </layout>
  </widget>
  <action name="actionAddToTimeline">
   <property name="text">
    <string>AddToTimeline</string>
   </property>
   <property name="toolTip">
    <string>Add To Timeline</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EffectListView</class>
   <extends>QListView</extends>
   <header>effectlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
